packages/api/src/transforms.ts

```diff
@@ -35,3 +35,26 @@ export function normalizeRequest(event: APIGatewayProxyEvent): Request {
     body,
   }
 }
+
+// Internal note:  Equivalent to dnull package on npm, which seems to have import issues in latest versions
```

Comment:
I got a build shipped for this: https://github.com/brielov/dnull/issues/65

Reply:
Thanks Orta! We might switch to dnull internally and keep the removeNulls export